src/main/java/web/pages/flipkart/ProductPage.java

```diff
@@ -17,4 +22,13 @@ public CartPage addToCartShoe(){
         addToCart.click();
         return new CartPage();
     }
+
+    public void verifyBackToTop(){
```

Comment:
not sure why you have named this verify....? I do not see this function verifying anything